bindings/python/Jamfile

```diff
@@ -31,16 +33,26 @@ if $(LIBTORRENT_PYTHON_INTERPRETER)
 # copied from boost 1.63's boost python jamfile
 rule find-py3-version
 {
-    local versions = [ feature.values python ] ;
-    local py3ver ;
-    for local v in $(versions)
-    {
-        if $(v) >= 3.0
-        {
-            py3ver = $(v) ;
-        }
-    }
-    return $(py3ver) ;
+	local py3ver ;
+	# expect the path to be something like boost_1_68_0
```

Comment:
Perhaps avoid using path, it seems fragile and a workflow I have used renamed boost directory

What about parsing version.cpp?

Reply:
yeah, that was my next thought

Reply:
Can you `import boost` and get the version?